new/analyse/widen-dependency-ranges.ts

```diff
@@ -0,0 +1,32 @@
+import type { Module } from '../types';
+
+import { clone } from 'lodash';
```

Comment:
Please, decide for one library only. https://github.com/xing/rlsr/blob/master/new/analyse/create-dependency-tree.ts#L2. I'd suggest Ramda. BTW: Lodash creates a vulnerability warning although I've just updated dependencies,

Reply:
Oops. Probably VS autocomplete. I'll fix this